### ISSUE TITLE
Compile Error

### ISSUE THREAD

**Issue by MandrewM** [luc-github/Repetier-Firmware-0.92#206]
I just downloaded the latest version to re-flash my DaVinci 1.0. When Compiling the latest (16/08/29) version from the master branch there is an error in the HAL.h file. I have included a screen shot of the error I received. I will be re-flashing with an older version (15/06/04) that does not have this error.
<img width="960" alt="compile error" src="https://cloud.githubusercontent.com/assets/12077628/19837645/f19e13b8-9e8b-11e6-82d0-ce97c85e5c95.PNG">

**Comment by luc-github** [comment]
Hi did you check the FAQ? looks like you use old version of IDE

**Comment by MandrewM** [comment]
I will try updating my IDE to the new version.  I thought the problem was from the function SetInterruptPriority() not being defined elsewhere in Repetier. I guess this is built in to the updated IDE version? I am not very familiar with this code. Thanks!

**Comment by luc-github** [comment]
it is a method from Serial which is arduino not repetier

**Comment by Agurt** [comment]
Hello,

I seem to be having a similar problem except I'm using the correct version. Any help would be greatly appreciated. 
<img width="1280" alt="screen shot 2016-11-05 at 3 42 14 pm" src="https://cloud.githubusercontent.com/assets/10872407/20034059/a6ee787a-a36e-11e6-9a56-535d76defcd0.png">

**Comment by luc-github** [comment]
yes this different issue,  I guess happen after a module update, close ide and open again
if issue occur again remove your due module close ide open again and reinstall the  close ide then open again and  compi!e if ok then modify variant.cpp usbcore.cpp
if issue happen again uninstall everything and eventually delete due modules files manully
this is pure arduino issue not related to fw

**Comment by Agurt** [comment]
Thanks for the help. A friend of mine came over and we ended up figuring out that IDE 1.6.9 isn't compatible with OSX Sierra. We switched to 1.6.12 and are able to upload fw to the board now.

I'm getting this error from repetier now though: 
`fatal:Heater/sensor failure Printer stopped and heaters disabled due to this error. Fix error and restart with M999.`

Could this have something to do with using IDE 1.6.12?
